frontend/pages/projects/_id/affective-annotation/index.vue

```diff
@@ -371,6 +353,20 @@ export default {
     this.relationTypes = await this.$services.relationType.list(this.projectId)
     this.project = await this.$services.project.findById(this.projectId)
     this.progress = await this.$services.metrics.fetchMyProgress(this.projectId)
+
+    const scaleTypesRaw = await this.$services.scaleType.list(this.projectId)
+    if (scaleTypesRaw.length > 0) {
+      const scaleTypesIdsTexts = {}
+      const scaleTypesTextsIds = {}
+      scaleTypesRaw.forEach(function(item) {
+        scaleTypesIdsTexts[item.id] = item.text
+        scaleTypesTextsIds[item.text] = item.id
+      })
+      this.scaleTypesIdsTexts = scaleTypesIdsTexts
+      this.scaleTypesTextsIds = scaleTypesTextsIds
+    } else {
+      alert("No scale labels found. Please import scale labels first!")
```

Comment:
1. I agree, ideally the JSON file containing the scale labels should be available somewhere in `frontend` directory so that it can be imported to the db automatically and also the same file can be used in `index.vue`. In that case, the mapping can refer to the content of the file and everything will be more-or-less fine XD Do you think I should just add the JSON file in this PR?
2. Yeah XD I just use alert as a temporary solution for now..... And it is not really good also because the Summary mode doesn't need scale labels, but the alert will appear too. XD Maybe we can add an additional variable like `isScaleImported`, and if the value is false then the annotation box won't be shown, do you like that approach?

I think I will be able to make further adjustments on Monday 😅